src/brain_teasers/python/design_browser_history/README.md

```diff
@@ -0,0 +1,81 @@
+
+---
+##BrowserHistory Class
```

Comment:
minor issue. This title isn't showing up formatted because you need space between the hashtags and the title. like  : `## BrowserHistory Class` instead of `##BrowserHistory Class`

Reply:
other than that LGTM ( === looks good to me)